tornado/curl_httpclient.py

```diff
@@ -509,6 +509,9 @@ def ioctl(cmd: int) -> None:
         if request.client_key is not None:
             curl.setopt(pycurl.SSLKEY, request.client_key)
 
+        if request.ssl_cipher is not None:
+            curl.setopt(pycurl.SSL_CIPHER_LIST, request.ssl_cipher)
```

Comment:
Instead of introducing a new curl-only option, could we just read it from the SSLContext?  Something like this:

    if request.ssl_options is not None:
        curl.setopt(pycurl.SSL_CIPHER_LIST, request.ssl_options.get_ciphers())

It might be a little weird to have only parts of the ssl_options be supported by curl_httpclient, but maybe this could be the start of supporting more parts of it.

Reply:
Supporting only a part of ssl_options sound like a little confused, but it's better than add more arguments if in future supporting more parts. So, i'm gonna change this code and send other PR.